src/app/TestStatusTable.tsx

```diff
@@ -11,9 +13,10 @@ import {
 import { Table, Thead, Tr, Th, Tbody, Td } from "@patternfly/react-table";
 import { repoStatus } from "@/getters";
 import { useSession } from "next-auth/react";
+import { MoonIcon, SunIcon } from '@patternfly/react-icons';
 import { LoginButton } from "../components";
 
-import "@patternfly/react-core/dist/styles/base.css";
+import "@patternfly/patternfly/patternfly.css";
```

Comment:
This may be known but just in case
* `@patternfly/patternfly/patternfly.css` is all patternfly styles except for utility classes
* `@patternfly/react-core/dist/styles/base.css` is just our global styles (no component/layout styles) minus font awesome and possibly other stuff that happens via [patternfly-react/packages/react-core/scripts/copyStyles.mjs](https://github.com/patternfly/patternfly-react/blob/96e94dffab4bd6d645de82e955520cf62cdf0053/packages/react-core/scripts/copyStyles.mjs)
 
So I'd want to make sure that with this update, styles aren't being pulled in twice/multiple times (once from `patternfly.css` and again from an individual component stylesheet - for example, `button.css` which will be imported when you `import { Button } from "patternfly/react-core"`. You could see if that's the case by inspecting a button and seeing if you have the same button styles listed multiple times in dev tools with the duplicate styles crossed out.

And for reference:
* patternfly.css is about 1.5mb (you can use patternfly.min.css, it's 1.4mb)
* patternfly-base.css is about 184k